src/Infrastructure/MembershipApplicationTrackingInfo.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace WMDE\Fundraising\Frontend\Infrastructure;
+
+/**
+ * @licence GNU GPL v2+
+ * @author Jeroen De Dauw < jeroendedauw@gmail.com >
+ */
+class MembershipApplicationTrackingInfo {
+
+	private $totalImpressionCount;
+	private $singleBannerImpressionCount;
+	private $confirmationPage;
+	private $confirmationPageCampaign;
```

Comment:
> The latter two can/should be combined into one (campaign/keyword).

Can we do that easily, or is that an improvement we better hold off until we have the app functional?